### PR TITLE
amass-intel: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/common/amass-intel.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# amass intel
+
+> Recopila información de código abierto sobre una organización, como dominios raíz y ASNs.
+> Más información: <https://github.com/OWASP/Amass/blob/master/doc/user_guide.md#the-intel-subcommand>.
+
+- Busca dominios raíz en un rango de direcciones IP:
+
+`amass intel -addr {{192.168.0.1-254}}`
```
Comment (EmilySeville7cfg) [inline pages.es/common/amass-intel.md:8]:
```suggestion
- Encuentre dominios raíz en un rango de direcciones IP específico:

`amass intel -addr {{192.168.0.1-254}}`
```

Optional fix (error source: original page): `specific` may sound better.

File: pages.es/common/amass-intel.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Utiliza métodos activos de reconocimiento:
+
+`amass intel -active -addr {{192.168.0.1-254}}`
+
+- Busca dominios raíz relacionados con un dominio:
+
+`amass intel -whois -d {{nombre_de_dominio}}`
```
Comment (EmilySeville7cfg) [inline pages.es/common/amass-intel.md:16]:
```suggestion
- Encuentre dominios raíz relacionados con un dominio específico:

`amass intel -whois -d {{nombre_de_dominio}}`
```

The same issue as above.

File: pages.es/common/amass-intel.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Busca ASNs pertenecientes a una organización:
+
+`amass intel -org {{nombre_de_organizacion}}`
```
Comment (EmilySeville7cfg) [inline pages.es/common/amass-intel.md:20]:
```suggestion
- Encuentre ASN pertenecientes a una organización específica:

`amass intel -org {{nombre_de_organizacion}}`
```

The same issue as above.

File: pages.es/common/amass-intel.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- Busca dominios raíz pertenecientes a un determinado número de sistemas autónomos:
+
+`amass intel -asn {{asn}}`
```
Comment (EmilySeville7cfg) [inline pages.es/common/amass-intel.md:24]:
```suggestion
- Encuentre dominios raíz pertenecientes a un Número de Sistema Autónomo específico:

`amass intel -asn {{string}}`
```

Almost the same issue as above.
Optional fix: `asn` term duplication can be reduced.
Warning: failed to deduce value type of `-asn` option, `string` used as a fallback.

File: pages.es/common/amass-intel.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- Guarda los resultados en un archivo de texto:
+
+`amass intel -o {{archivo_de_salida}} -whois -d {{nombre_de_dominio}}`
```
Comment (EmilySeville7cfg) [inline pages.es/common/amass-intel.md:28]:
```suggestion
- Guarde los resultados en un archivo de texto específico:

`amass intel -o {{ruta/al/archivo_de_salida}} -whois -d {{nombre_de_dominio}}`
```

Optional fix (error source: original page): `specific` may sound better.
Mandatory fix (error source: original page): [path placeholder is broken](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths).

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
Mandatory fixes found, changes requested.
Thanks for a contribution! ❤️

**Review by EmilySeville7cfg — APPROVED** [review]

File: pages.es/common/amass-intel.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+# amass intel
+
+> Recopila información de código abierto sobre una organización, como dominios raíz y ASNs.
+> Más información: <https://github.com/OWASP/Amass/blob/master/doc/user_guide.md#the-intel-subcommand>.
+
+- Encuentre dominios raíz en un rango de direcciones IP específico:
```
Comment (navarroaxel) [inline pages.es/common/amass-intel.md:6]:
```suggestion
- Encuentra dominios raíz en un rango de direcciones IP específico:
```

File: pages.es/common/amass-intel.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+`amass intel -addr {{192.168.0.1-254}}`
+
+- Utiliza métodos activos de reconocimiento:
+
+`amass intel -active -addr {{192.168.0.1-254}}`
+
+- Encuentre dominios raíz relacionados con un dominio específico:
```
Comment (navarroaxel) [inline pages.es/common/amass-intel.md:14]:
```suggestion
- Encuentra dominios raíz relacionados con un dominio específico:
```

File: pages.es/common/amass-intel.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+`amass intel -whois -d {{nombre_de_dominio}}`
+
+- Encuentre ASN pertenecientes a una organización específica:
```
Comment (navarroaxel) [inline pages.es/common/amass-intel.md:18]:
```suggestion
- Encuentra ASN pertenecientes a una organización específica:
```

File: pages.es/common/amass-intel.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+`amass intel -org {{nombre_de_organizacion}}`
+
+- Encuentre dominios raíz pertenecientes a un Número de Sistema Autónomo específico:
```
Comment (navarroaxel) [inline pages.es/common/amass-intel.md:22]:
```suggestion
- Encuentra dominios raíz pertenecientes a un Número de Sistema Autónomo específico:
```

File: pages.es/common/amass-intel.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+
+`amass intel -asn {{string}}`
+
+- Guarde los resultados en un archivo de texto específico:
```
Comment (navarroaxel) [inline pages.es/common/amass-intel.md:26]:
```suggestion
- Guarda los resultados en un archivo de texto específico:
```

File: pages.es/common/amass-intel.md [hunk 11]
```diff
@@ -0,0 +1,28 @@
+# amass intel
+
+> Recopila información de código abierto sobre una organización, como dominios raíz y ASNs.
+> Más información: <https://github.com/OWASP/Amass/blob/master/doc/user_guide.md#the-intel-subcommand>.
+
+- Encuentre dominios raíz en un rango de direcciones IP específico:
+
+`amass intel -addr {{192.168.0.1-254}}`
+
+- Utiliza métodos activos de reconocimiento:
```
Comment (navarroaxel) [inline pages.es/common/amass-intel.md:10]: I prefer the verb `usar`
```suggestion
- Usa métodos activos de reconocimiento:
```

**Review by kbdharun — APPROVED** [review]
LGTM after navarroaxel's suggestions

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**